index.js

```diff
@@ -1,15 +1,32 @@
-var FluxConstant = function FluxConstant(name) {
+var FluxConstant = function FluxConstant(name, id) {
 
-    this.name = name;
+    this.id = String(id || (FluxConstant.lastId += 1));
```

Comment:
how about if instead of using a `lastId`, we use the value provided by [`Date.now()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now) instead?